app/Views/errors/html/debug.css

```diff
@@ -65,13 +64,9 @@ p.lead {
     display: inline;
 }
 
-.footer {
+.environment {
     background: var(--dark-bg-color);
     color: var(--light-text-color);
```

Comment:
Can you give space padding text to background? like a before fixed but change to proportional. To be honest i want in flying bottom like debug bar

Reply:
Added `padding: 0.2rem;`:
![Screenshot 2024-03-15 11 57 39](https://github.com/codeigniter4/CodeIgniter4/assets/87955/409e5c48-15f5-49b6-a2e3-f61711e98c71)